updating/updating-restricted-network-cluster/mirroring-image-repository.adoc

```diff
@@ -0,0 +1,169 @@
+== Mirroring resources using the oc-mirror plugin
+
+You can use the oc-mirror OpenShift CLI (`oc`) plugin to mirror images to a mirror registry in your fully or partially disconnected environments. You must run oc-mirror from a system with internet connectivity in order to download the required images from the official Red Hat registries.
+
+The following steps outline the high-level workflow on how to use the oc-mirror plugin to mirror images to a mirror registry:
+
+. Create an image set configuration file.
+. Mirror the image set to the mirror registry by using one of the following methods:
+** Mirror an image set directly to the mirror registry.
+** Mirror an image set to disk, transfer the image set to the target environment, then upload the image set to the target mirror registry.
+. Install the `ImageContentSourcePolicy` and `CatalogSource` resources that were generated by oc-mirror into the cluster.
+. Repeat these steps to update your mirror registry as necessary.
+
+// About the oc-mirror plugin
+include::modules/oc-mirror-about.adoc[leveloffset=+2]
+
+// oc-mirror compatibility and support
+include::modules/oc-mirror-support.adoc[leveloffset=+2]
+
+// About the mirror registry
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For information about viewing the CRI-O logs to view the image source, see xref:../../installing/validating-an-installation.adoc#viewing-the-image-pull-source_validating-an-installation[Viewing the image pull source].
+
+// Installing the oc-mirror OpenShift CLI plugin
+include::modules/oc-mirror-installing-plugin.adoc[leveloffset=+2]
+
+// Creating the image set configuration
+include::modules/oc-mirror-creating-image-set-config.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#oc-mirror-imageset-config-params_mirroring-ocp-image-repository[Image set configuration parameters]
+* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#oc-mirror-image-set-examples_mirroring-ocp-image-repository[Image set configuration examples]
+* xref:../../updating/updating-restricted-network-cluster/restricted-network-update-osus.adoc#update-service-overview_updating-restricted-network-cluster-osus[About the OpenShift Update Service]
+
+[id="mirroring-image-set"]
+=== Mirroring an image set to a mirror registry
+
+You can use the oc-mirror CLI plugin to mirror images to a mirror registry in a xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#mirroring-image-set-partial[partially disconnected environment] or in a xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#mirroring-image-set-full[fully disconnected environment].
+
+These procedures assume that you already have your mirror registry set up.
```

Comment:
I think that sentence (which originally comes from the install section's oc-mirror page) is saying "you should have followed everything from the beginning of this page up until this point". And the specific prereqs for each of procedures in this section already refer to the individual tasks you'd accomplish if you did follow everything up until this point of the page. So it might actually come out redundant and confusing, though I'm not sure.

@bergerhoffer adding this as an extra prereq would result in changes to your oc-mirror modules. Do you have an opinion/are you okay with me adding it to your modules?